packages/protocol/foundry.toml

```diff
@@ -3,7 +3,7 @@ src = 'src'
 out = 'out'
 libs = ['lib']
 optimizer = true
-optimizer_runs = 10000
+optimizer_runs = 1000
```

Comment:
should we use the default 200?

Reply:
This is how optimization_runs works. The higher the optimization_runs the cheaper gas cost to call functions for users, but the higher the contract bytecode. The lower the optimization runs the higher the gas cost for users, but lower the contract bytecode. For the user benefit, you try to run as high optimizations as you can, as the gas cost of your contract will be paid for as long as the contract is used.